work.html

```diff
@@ -64,10 +56,48 @@ <h2>We're not currently hiring, but...</h2>
 
 </section>
 
+
 <section>
-  <h2>Previous roles</h2>
+  <a name="mini-apprenticeship"></a>
+  <h3>Mini apprenticeship (6 weeks)</h3>
+
+  <p>Position Dev is piloting a mini apprenticeship program! We’re looking for a junior developer to join us for 6 weeks
+  on-site at our New York City office to work on the Wayland magazine platform, which is used by Jacobin, Current
+  Affairs,and Monthly Review.</p>
+
+  <h4>What you’ll be doing</h4>
+
+  <p>You’ll create and style web pages with HTML, CSS, and JavaScript. You’ll learn how to build a web app with Haskell,
+  and  get the chance to contribute to open-source Haskell libraries. You’ll solve real-world problems that affect users, and
```

Comment:
I think we can remove the "get the chance". From the upcoming tickets, I'll need to to make changes to Larceny and Offset, so this is def going to happen. :)